### PR TITLE
cromwell 62

### PR DESCRIPTION
**Pull request by kshakir**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Review by Rylan12 — COMMENTED** [review]
Thanks! Can you please move the `bottle :unneeded`?

File: Formula/cromwell.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Cromwell < Formula
   desc "Workflow Execution Engine using Workflow Description Language"
   homepage "https://github.com/broadinstitute/cromwell"
-  url "https://github.com/broadinstitute/cromwell/releases/download/61/cromwell-61.jar"
-  sha256 "badffaa34faf92b723fb04046169a35921a547a2e3cc49f2fa993377397752f8"
+  url "https://github.com/broadinstitute/cromwell/releases/download/62//cromwell-62.jar"
```
Comment (nandahkrishna) [inline Formula/cromwell.rb:4]:
```suggestion
  url "https://github.com/broadinstitute/cromwell/releases/download/62/cromwell-62.jar"
```

File: Formula/cromwell.rb [hunk 2]
```diff
@@ -15,8 +15,8 @@ class Cromwell < Formula
   depends_on "openjdk"
 
   resource "womtool" do
-    url "https://github.com/broadinstitute/cromwell/releases/download/61/womtool-61.jar"
-    sha256 "0b5872f6400a932d43766fe158e51150e890a0471bb129ee99b96a916a9eff40"
+    url "https://github.com/broadinstitute/cromwell/releases/download/62//womtool-62.jar"
```
Comment (nandahkrishna) [inline Formula/cromwell.rb:18]:
```suggestion
    url "https://github.com/broadinstitute/cromwell/releases/download/62/womtool-62.jar"
```

**Review by nandahkrishna — COMMENTED** [review]
Apart from removing the `bottle :unneeded`, some minor changes.

**Review by nandahkrishna — APPROVED** [review]
`568803dcb37d67c83d5106b1b61ce3649d6277ad5fd97e83e8d340b7b45ab5bc` on all platforms. Thanks @kshakir!

**Pull request closed without merging**